armadillo/src/main/java/org/molgenis/armadillo/controller/DataController.java

```diff
@@ -125,39 +135,43 @@ public ResponseEntity<Void> tableExists(
       summary = "Load table",
       description = "Load a table",
       security = {@SecurityRequirement(name = "jwt")})
+  @ApiResponses(
+      value = {
+        @ApiResponse(responseCode = "200", description = "Object loaded successfully"),
+        @ApiResponse(
+            responseCode = "404",
+            description = "Unknown project or object",
+            content = @Content(mediaType = "application/json")),
+        @ApiResponse(
+            responseCode = "401",
+            description = "Unauthorized",
+            content = @Content(mediaType = "application/json"))
+      })
   @PostMapping(value = "/load-table")
   public CompletableFuture<ResponseEntity<Void>> loadTable(
       Principal principal,
       @Valid @Pattern(regexp = SYMBOL_RE) @RequestParam String symbol,
       @Valid @Pattern(regexp = TABLE_RESOURCE_REGEX) @RequestParam String table,
       @Valid @Pattern(regexp = SYMBOL_CSV_RE) @RequestParam(required = false) String variables,
```

Comment:
@clemens-tolboom You  mean either of those?
![Screenshot 2024-02-12 at 15 30 43](https://github.com/molgenis/molgenis-service-armadillo/assets/10075439/2ecf5ce7-1d5f-49a8-9fe2-8968afad4079)
![Screenshot 2024-02-12 at 15 31 48](https://github.com/molgenis/molgenis-service-armadillo/assets/10075439/e56ebac3-2bc3-486b-8332-5ebe5bef9aac)

You should select variables, otherwise you wouldn't have to create a view, then you can just give access to the original, so I think this is fine. If you do want to link to the complete table, I guess you can just provide all variables. In the API/swagger that's probably annoying, but in the UI we can fix that.